src/main/java/org/checkerframework/specimin/PrunerVisitor.java

```diff
@@ -34,6 +34,7 @@
 import java.util.Iterator;
 import java.util.Optional;
 import java.util.Set;
+import javax.annotation.Nullable;
```

Comment:
Please use `org.checkerframework.checker.nullness.qual`'s `@Nullable` annotation instead of the deprecated `javax.annotation.Nullable`.